[2024-09-02T01:55:54.897+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T01:55:54.916+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: localIngestionDag.scrape manual__2024-09-02T00:55:35.799937+00:00 [queued]>
[2024-09-02T01:55:54.930+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: localIngestionDag.scrape manual__2024-09-02T00:55:35.799937+00:00 [queued]>
[2024-09-02T01:55:54.932+0000] {taskinstance.py:2856} INFO - Starting attempt 16 of 16
[2024-09-02T01:55:54.953+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): scrape> on 2024-09-02 00:55:35.799937+00:00
[2024-09-02T01:55:54.961+0000] {standard_task_runner.py:72} INFO - Started process 76 to run task
[2024-09-02T01:55:54.966+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'localIngestionDag', 'scrape', 'manual__2024-09-02T00:55:35.799937+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpq3f54fof']
[2024-09-02T01:55:54.970+0000] {standard_task_runner.py:105} INFO - Job 37: Subtask scrape
[2024-09-02T01:55:55.039+0000] {task_command.py:467} INFO - Running <TaskInstance: localIngestionDag.scrape manual__2024-09-02T00:55:35.799937+00:00 [running]> on host 0cd03fc143e7
[2024-09-02T01:55:55.156+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='localIngestionDag' AIRFLOW_CTX_TASK_ID='scrape' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T00:55:35.799937+00:00' AIRFLOW_CTX_TRY_NUMBER='16' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T00:55:35.799937+00:00'
[2024-09-02T01:55:55.158+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T01:55:55.837+0000] {logging_mixin.py:190} INFO - Downcasting floats.
[2024-09-02T01:55:56.094+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/scrapeNFLData.py", line 16, in scrape_weekly_data
    print('saved to db. Num cols = ' + num_columns)
TypeError: can only concatenate str (not "int") to str
[2024-09-02T01:55:56.109+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=localIngestionDag, task_id=scrape, run_id=manual__2024-09-02T00:55:35.799937+00:00, execution_date=20240902T005535, start_date=20240902T015554, end_date=20240902T015556
[2024-09-02T01:55:56.130+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T01:55:56.132+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 37 for task scrape (can only concatenate str (not "int") to str; 76)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/airflow/dags/scrapeNFLData.py", line 16, in scrape_weekly_data
    print('saved to db. Num cols = ' + num_columns)
TypeError: can only concatenate str (not "int") to str
[2024-09-02T01:55:56.185+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2024-09-02T01:55:56.221+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-02T01:55:56.229+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
